jadx-gui/src/main/java/jadx/gui/treemodel/JMethod.java

```diff
@@ -90,7 +91,11 @@ String makeBaseString() {
 
 	@Override
 	public String makeString() {
-		return UiUtils.typeFormat(makeBaseString(), getReturnType());
+		return UiUtils.typeFormatHtml(makeBaseString(), getReturnType());
```

Comment:
Should this be just a `typeFormat` without `Html`?
```suggestion
		return UiUtils.typeFormat(makeBaseString(), getReturnType());
```
And also overriden makeStringHtml() method:
```java
	@Override
	public String makeStringHtml() {
		return UiUtils.typeFormatHtml(makeBaseString(), getReturnType());
	}
```

Reply:
Looks like I forgot to adapt those methods. Should be addressed in the latest commit.